### ISSUE TITLE
Possible redundancy?

### ISSUE THREAD

**Issue by Funtime60** [jfarseneau/antennas#113]
I was rebuilding my TVHeadEnd setup today and I found this section in the settings. Is it possible that this functionality has been integrated into TVHeadEnd? I'm having trouble finding mentions of this with my cursory google search. I keep finding dead forum links and feature requests from 2015 and 2018 so I have to assume that it doesn't work or something?

![image](https://github.com/jfarseneau/antennas/assets/24595923/b2a373d2-181a-4f8a-9f00-6d48644651be)

**Comment by jatkins11** [comment]
I ended up finding those settings in TVHE as well. Unfortunately I don't believe they work or if they do then I was unable to configure it properly.

**Comment by Jimbob-B** [comment]
Yeah, I had a play with this on my Pi using build number 4.3-2299 from their Git and couldn't get it to work either. Searching for documentation didn't reveal much apart from a long thread on their bug tracker that didn't describe the functionality (at a quick glance).
